apps/utils.py

```diff
@@ -136,3 +140,18 @@ def serialize(self, queryset, *, stream=None, fields=None, use_natural_foreign_k
                 self.first = False
         self.end_serialization()
         return self.getvalue()
+
+
+def send_templated_email(template=None, subject=None, context=None, sender=None, recipients=None):
+    if not template or not subject or not recipients:
+        raise ValidationError
+
+    if not isinstance(recipients, list):
+        raise ValidationError('recipients should be a list')
```

Comment:
The recipients can technically be a list or tuple for Django's built in mail system, though that is a minor change that might not be necessary